# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/sessionstore/moz.build
# Commit: 2a8633af0279
# Full Hash: 2a8633af0279379685ba781c6c469e8b07ec13f4
# Author: Mathew Hodson <mathew.hodson@gmail.com>
# Date: 2022-03-26 09:28:42
# Regressor Bug: 1752853
# File Overlap Count: 1
# Description:
#   Bug 1752853 - Stop using a worker to write session store. r=Gijs,Standard8
#   
#   Remove now unused pref "browser.sessionstore.max_write_failures".
#   Rename and simplify test_shutdown_cleanup.js to make it more clear that
#   it is testing the isFinalWrite option.
# ==============================================================================

diff -r dd81d58d9d5f -r 2a8633af0279 browser/components/sessionstore/moz.build
--- a/browser/components/sessionstore/moz.build	Fri Mar 25 19:19:15 2022 +0000
+++ b/browser/components/sessionstore/moz.build	Fri Mar 25 19:42:36 2022 +0000
@@ -21,8 +21,7 @@
     "SessionSaver.jsm",
     "SessionStartup.jsm",
     "SessionStore.jsm",
-    "SessionWorker.js",
-    "SessionWorker.jsm",
+    "SessionWriter.jsm",
     "StartupPerformance.jsm",
     "TabAttributes.jsm",
     "TabState.jsm",